sdcm/tester.py

```diff
@@ -796,6 +796,11 @@ def is_encrypt_keys_needed(self):
         )
 
     def prepare_kms_host(self) -> None:
+        if self.params.is_enterprise and self.params.get('cluster_backend') == 'aws' and not self.params.get('scylla_encryption_options'):
+            self.params['scylla_encryption_options'] = \
```

Comment:
Please, do not use backslashes.
Please, keep key and value on the same lines.

Reply:
keeping it all in the same line, it becomes a bit long...
no sure why to insist on it. but I don't care.